content/100-getting-started/01-quickstart.mdx

```diff
@@ -155,4 +155,4 @@ Or perhaps you're looking at **starting a new project with Prisma**? The [Start
 
 Have you made something cool with Prisma? Use the hashtag **#MadeWithPrisma** and let us know!
 
-</Admonition>
\ No newline at end of file
+</Admonition>
```

Comment:
What is the diff here? GH UI does not show anything useful.

Reply:
Looks like it adds missing `EOF`

<img width="635" alt="image" src="https://user-images.githubusercontent.com/4734297/124756503-f325a480-df45-11eb-8054-820fd1ec940f.png">